docs/design-proposals/multi-tenancy/multi-tenancy-network.md

```diff
@@ -238,6 +241,17 @@ Network object specifies service IPAM as *external*:
       service:
         ipam: external
 
+#### User hints
+
+User can provide hints about service IP address.
+
+For Arktos managed pool, it will look at the specific annotation of service object like below
+```arktos.futurewei.com/vip-hint: {"cidr":"10.10.1.0/26"}```
```

Comment:
Is this the CIDR range for VPC or subnet? 
Passing in a CIDR range is not very helpful, as two VPCs or two subnets can have the same CIDR. We should track using name of the subnet or vpc.

Reply:
this cidr in vip-hint is for Arktos service objects only. Using vip-hint to convey optional service intent is optional. If Mizar don't take cidr or ip hint, it is totally fine to ignore. Not to implement the Guaranteed  policy semantics is OK, too. We can put it explicit that first version only supports besteffort.

Reply:
Correct me if I am wrong: Mizar needs to launch a pod with IP address that is within CIDR 10.10.1.0/26 range. 
However, the subnet field is optional. From pure network perspective, a pod needs to be launched within a subnet. So if we allow using to specify a CIDR range that does not reference to an existing subnet, we might assign an IP address that does not fall into range provided by the user here (in this case is 10.10.1.0/26) 
For example, if vpc-1 has only one subnet with CIDR 10.10.2.0/24 and user assigned a vip-hint of cidr = 10.10.1.0/26. 
Mizar only reads the subnet's CIDR range and pick an IP address from 10.10.2.0/24 pool. 
Will this cause an error?

Reply:
in that case, mark network-readiness to error,  as it cannot be satisfied. Meanwhile, please fire an event to Arktos reporting the error.